app/Http/Controllers/SchoolController.php

```diff
@@ -0,0 +1,109 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Models\School;
+use Illuminate\Http\Request;
+use Illuminate\Support\Str;
+
+class SchoolController extends Controller
+{
+
+    public function uuid()
+    {
+        return Str::uuid()->toString();
+    }
+    
+    /**
+     * Display a listing of the resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function index()
+    {
+        //
+    }
+
+    /**
+     * Show the form for creating a new resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function create(Request $request)
+    {
+        $id = $this->uuid();
+        $schoolName = $request->get('schoolName');
+        $schoolCode = $request->get('schoolCode');
+        $status = 'Active';
```

Comment:
We will need this status thingy to be  an enum, Active or Inactive. But yes default should be active

Reply:
It's an enum, you'll see from the db migration. Front end will send the value

Reply:
Seen.